source/_integrations/xiaomi_miio.markdown

```diff
@@ -1635,6 +1636,22 @@ automation:
           segments: [1, 1]
 ```
 
+### Service `xiaomi_miio.vacuum_reset_consumable`
```

Comment:
As per comment on the linked Core PR. These should become buttons, not services.